crates/rspack_fs/src/native.rs

```diff
@@ -57,6 +57,12 @@ cfg_async! {
       let fut = async move { tokio::fs::write(file, data).await.map_err(Error::from) };
       Box::pin(fut)
     }
+
+    fn remove_dir_all<P: AsRef<Path>>(&self, dir: P) -> BoxFuture<'_, Result<()>> {
+      let dir = dir.as_ref().to_string_lossy().to_string();
+      let fut = async move { tokio::fs::remove_dir_all(dir).await.map_err(Error::from) };
```

Comment:
I believe [tokio::fs::remove_dir_all](https://docs.rs/tokio/0.3.1/tokio/fs/fn.remove_dir_all.html) is ok to accept `Path` as arguments. You should be able to just pass `dir` `to tokio::fs::remove_dir_all`.